src/commands/check/index.ts

```diff
@@ -97,16 +162,34 @@ export default class Check extends SwankyCommand<typeof Check> {
       },
       {
         title: "Verify ink version",
```

Comment:
This becomes "Verify cargo contract version". The goal is to ensure a min cargo contract version is installed for the higher ink version used in contracts. This is ensured with `CARGO_CONTRACT_INK_DEPS`.

I think you can do something like this with semver:

```
const compatibleInkVersion = cargoContractInkDependencies.find(dep =>
        semver.satisfies(cargoContractVersion, `>=${dep.minCargoContractVersion}`)
      )?.inkVersion;
```